loginserver/options.h

```diff
@@ -142,6 +152,7 @@ class Options {
 	bool        world_trace;
 	bool        dump_in_packets;
 	bool        dump_out_packets;
+	bool		display_expansions;
```

Comment:
Replace the tab with spaces between the `bool` and `display_expansions` here for alignment

Reply:
Done